tools/maxquant/maxquant.xml

```diff
@@ -364,11 +311,11 @@ sample:$j
     <expand macro="misc_conditional" />
   </inputs>
   <outputs>
-    <data format="tabular" name="output_protein_groups" label="MaxQuant Protein Groups for ${on_string}"/>
-    <data format="tabular" name="output_peptides" label="MaxQuant Peptides for ${on_string}"/>
-    <data format="tabular" name="output_evidence" label="MaxQuant Evidence for ${on_string}"/>
-    <data format="tabular" name="output_parameters" label="MaxQuant Tabular Parameters for ${on_string}"/>
-    <data format="tabular" name="output_msms" label="MaxQuant MSMS for ${on_string}"/>
+    <data format="csv" name="output_protein_groups" label="MaxQuant Protein Groups for ${on_string}"/>
```

Comment:
Is a comma separated files needed for downstream analysis? If not can we produce tabular files as output by simply piping all outputs through `sed`? Keeping everything `tab` separated as a big advantage in Galaxy. You get a ton of tools for free.